JSDemos/Demos/Validation/Overview/description.md

```diff
@@ -1,9 +1,34 @@
-DevExtreme Data Editors provide a&nbsp;powerful and seamless way to&nbsp;perform data validation. In&nbsp;this demo, editors are collected in&nbsp;an&nbsp;HTML&nbsp;form. Each editor is&nbsp;accompanied by&nbsp;the Validator component validating the editor against a&nbsp;set of&nbsp;specified rules. Editors are validated each time their value is&nbsp;changed. All validation errors are displayed in&nbsp;the ValidationSummary component.
+This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on the page with enabled data validation. In this demo, the editors are collected in an HTML form.
 
+To enable editor's data validation, you need to declare the [Validator](/Documentation/ApiReference/UI_Components/dxValidator/) component and implement the [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). You can attach several validation rules to one component. Each validation rule contains a rule type that must be one of the following:
```

Comment:
> To enable editor's data validation

Mind the articles, pretty please...

> You can attach **several** validation rules to one component.

I believed we've left this stuff in the past, yet here we are... "Several" is a colloquialism compared to "multiple". Moreover, they essentially mean exactly the same. "Several" is not smaller than "multiple". Yet "multiple" suits formal speech, such as documentation.

> Each validation rule contains a rule type that must be one of the following:

This is a very strange sentence. An awfully complicated way to express something that simple... Did you wanted to say the following:

> Each validation rule contains one of the following rule types:

However, the list below just enumerates all available validation rule types. Therefore, this sentence is still very strange.

> The following list contains all available validation rule types:

```suggestion
To data validation for an editor, you need to declare the [Validator](/Documentation/ApiReference/UI_Components/dxValidator/) component and implement [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). You can attach multiple validation rules to one component. The following list contains all available validation rule types:
```